syntax = "proto3";

package trt_sample.common;

message BackendParam {
    string onnx_file = 1;
    string engine_file = 2;
    repeated string input_names = 3;
    repeated string output_names = 4;
    int32 max_batch_size = 5;
};



